dist/tools/usb-serial/find-tty.sh

```diff
@@ -7,7 +7,7 @@
 exit_code=1
 
 for serial in "$@"; do
-    if ./ttys.py --format path --serial "$serial"; then
+    if ${RIOTTOOLS}/usb-serial/ttys.py --format path --serial "$serial"; then
```

Comment:
```suggestion
    if "${RIOTTOOLS}/usb-serial/ttys.py" --format path --serial "$serial"; then
```

Maybe we should also fall back `RIOTTOOLS` to `.` when not provided, so that `./find-tty.sh` still works?

Reply:
Sure I'll change.